src/GUI/efcpt/appsettings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  },
+  "NuGet": {
+    "RepositoryUri": "https://api.nuget.org/v3/index.json",
+    "PackageKey": "ErikEJ.EFCorePowerTools.CLI"
```

Comment:
What is this for? Also, the name is: ....Cli (casing)

Reply:
> What is this for? Also, the name is: ....Cli (casing)

I suppress this